src/builders/CommandStep.ts

```diff
@@ -65,19 +72,70 @@ export class CommandStep implements StepBuilder, KeyBuilder, LabelBuilder, Condi
     this.#plugins.push(plugin)
     return this
   }
+  
+  parallelism(parallelism: number): this {
+    this.#parallelism = parallelism
+    return this
+  }
+  
+  env(env: Record<string, string | number>): this {
+    this.#env = env
+    return this
+  }
 
-  build(): CommandStepSchema {
-    if (!this.#command) {
-      throw new Error('CommandStep requires a command.')
+  addEnv(key: string, value: string | number): this {
```

Comment:
thoughts around changing this to singular `env(key, value)` and removing the existing `env(object)` for consistency with the singular pattern?

Reply:
✅